buildSrc/src/main/kotlin/assertk.multiplatform.gradle.kts

```diff
@@ -130,3 +124,21 @@ val detektTest by tasks.registering(Detekt::class) {
 val detekt by tasks.getting {
     dependsOn(detektMain, detektTest)
 }
+
+// Node doesn't publish alpha versions for windows
+if(!DefaultNativePlatform.getCurrentOperatingSystem().isWindows) {
```

Comment:
`DefaultNativePlatform` is in an internal package, I think I used `Os.isFamily(Os.FAMILY_WINDOWS)` before

Reply:
That makes more sense 😅

I thought it was something like that. Shouldn't have trusted ChatGPT

Reply:
`Os.isFamily` comes from ant, which I guess isn't available here.

I found `OperatingSystem.current().isWindows` but that is also in an internal package. Seems like the only non internal way of doing this is by using `System.getProperty("os.name")` which is what they use under the hood anyways.

Reply:
> Os.isFamily comes from ant, which I guess isn't available here

It is, but I never had issues importing it in the past? Anyway using `System.getPropery("os.name")` is fine too.

Reply:
Guess my IDE was acting funny. Using the FQN worked, and now it's working with an import. Pushing change now.